S3/S3EventGridPoc/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Information": [
```

Comment:
Maybe move or copy this info to the README ? 
People could miss this file

Reply:
This is auto generated by lambda, nothing that I wrote